metaflow/plugins/project_decorator.py

```diff
@@ -25,6 +25,50 @@ class ProjectDecorator(FlowDecorator):
         Project name. Make sure that the name is unique amongst all
         projects that use the same production scheduler. The name may
         contain only lowercase alphanumeric characters and underscores.
+
+    MF Add To Current
```

Comment:
`MF Add To Current` - is there a special significance to this string?

Reply:
Yes, this is used to define a section to add fields to the current object. It's non standard and just our thing.